src/Features/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptSemanticChangeNotificationServiceWrapper.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.Notification;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
+{
+    internal readonly struct VSTypeScriptSemanticChangeNotificationServiceWrapper
+    {
+        private readonly ISemanticChangeNotificationService _underlyingObject;
+
+        public VSTypeScriptSemanticChangeNotificationServiceWrapper(ISemanticChangeNotificationService underlyingObject)
```

Comment:
What value is this if nothing can be read from it?

Reply:
Good catch. This is not used. Will remove.